### ISSUE TITLE
MCP tool property type

### ISSUE THREAD

**Issue by thomasdv** [inspector-apm/neuron-ai#266]
This is technically not a bug, but the very first MCP server I tried to use had a problem with this.

It listed a property without "type" - it was tool description. McpConnector then failed on this line in `createTool()` method, as `$prop['type']` was not defined:
```php
$type = PropertyType::fromSchema($prop['type']);
```

I know a property type should be defined, but maybe it could either default to string or skip such a property entirely? This should not cause an error.

Just a thought, thanks :)

**Comment by ilvalerione** [comment]
Hi @thomasdv, thank you for sharing. I think ignoring the property it's not a good isea, because the tool could even define it as required, and its invokation will fail eventually. Setting as string by default could be a good trade off.

**Comment by ilvalerione** [comment]
It will go live in the next release: https://github.com/inspector-apm/neuron-ai/blob/1.x/src/MCP/McpConnector.php#L112
